lib/StaticNarrative/exporter/app_processor.py

```diff
@@ -110,15 +112,16 @@ def _translate_param_value(
         value: None | int | str | list,
         param_spec: dict[str, Any],
         upas: dict[str, Any],
-    ):
-        """
+    ) -> list[Any] | str | int:
+        """Convert param values to forms that the SN can use.
+
         Overall flow.
         1. if value is a list, iterate everything below over it.
         2. inspect the spec to see what the value represents.
             a. if a
-        2. if value is a string, check if its an UPA, verify with the spec that it
+            a. if value is a string, check if it's an UPA, verify with the spec that it
             should be an object
-        3. if value is an int... stuff.
+            b. if value is an int... stuff.
```

Comment:
This looks a little off? Probably something I forgot to finish writing some years ago.